site/content/en/docs/administration/advanced/update_guide.md

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Update guide'
+linkTitle: 'Update guide'
+weight: 11
+description: 'Instructions for updating CVAT deployed with docker compose'
+---
+
+<!--lint disable heading-style-->
+
+## About CVAT data volumes
+
+Docker volumes are used to store all CVAT data:
```

Comment:
@azhavoro , could we show how to upgrade CVAT instance from v1.7.0 to v2.1.0? I belive in examples. It is easy to follow them. We will see (at least now), that the particular example works.

Reply:
Probably for the section it doesn't make sense. Should we add it in FAQ?